distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc

```diff
@@ -0,0 +1,94 @@
+:_content-type: ASSEMBLY
+[id="distr-tracing-tempo-configuring"]
+= Configuring and deploying distributed tracing platform Tempo
+include::_attributes/common-attributes.adoc[]
+:context: distr-tracing-tempo-configuring
+
+toc::[]
+
+The {TempoName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {TempoShortName} resources. You can either install the default configuration or modify the file to better suit your business requirements.
+
+{TempoName} has predefined deployment strategies. You specify a deployment strategy in the custom resource file. When you create a {TempoShortName} instance the Operator uses this configuration file to create the objects necessary for the deployment.
+
+.Tempo custom resource file showing deployment strategy
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simple
+  namespace: tempo-example
+spec:
+  storage:
+    secret:
+      # Name of the secret with the information to access the object storage
+      name: tempostack-odf
+      type: s3
+  storageSize: 1Gi
+  template:
+    queryFrontend:
+      jaegerQuery:
+        # Enable the Jaeger UI
+        enabled: true
+----
+
+The {TempoName} Operator currently supports the following deployment strategies:
+
+* *allInOne* (Default) - This strategy is intended for development, testing, and demo purposes; it is not intended for production use. The main backend components, Agent, Collector, and Query service, are all packaged into a single executable which is configured, by default. to use in-memory storage.
```

Comment:
IIRC We don't have strategies for tempo. I think we should remove this.

Reply:
Thank you. I'll remove it then.